compiler/testData/diagnostics/tests/funInterface/functionDelegateClashOnJvm.fir.kt

```diff
@@ -15,7 +15,7 @@ fun interface F3 {
     fun invoke()
 }
 
-fun interface F4 {
+<!FUN_INTERFACE_WRONG_COUNT_OF_ABSTRACT_MEMBERS!>fun<!> interface F4 {
     fun functionDelegate(): Function<*>? = null
```

Comment:
Also it's worth to implement `FunctionDelegateMemberNameClashChecker` in this PR, because it is part of checkers for `fun` interfaces